seq2science/rules/configuration_generic.smk

```diff
@@ -311,7 +337,8 @@ with FileLock(layout_cachefile_lock):
     config['layout'] = {**layout_cache,
                         **{k: v for k, v in config['layout'].items()},
                         **{k: v.get() for k, v in eutils_layout.items() if v.get() is not None},
-                        **{k: v.get() for k, v in trace_layout.items() if v.get() is not None}}
+                        **{k: v.get() for k, v in trace_layout1.items() if v.get() is not None},
+                        **{k: v.get() for k, v in trace_layout2.items() if v.get() is not None}}
```

Comment:
trace_layout2 overwrites trace_layout1, which overwrites eutils_layout, which overwrites the layout_cache. Is that intended?

Reply:
Yes that is intended (only overwriting when they are not None though). Trace layout has to overwrite eutils in case someone did an oopsie and wrote that their samples are single/paired end. Eutils is still necessary since it is the most robust method.